package.json

```diff
@@ -10,7 +10,7 @@
   "author": "NYPL",
   "license": "Apache-2.0",
   "dependencies": {
-    "simplified-registry-admin": "1.4.14"
```

Comment:
Update the version above as well.

Reply:
I think it is updated? See below!

Reply:
The registry admin package was updated but this app should be updated as well. In the changelog you have the new version as 1.0.11 so it should be updated on line 3 as well.

Reply:
Oh shoot!